src/app/components/builder-bricks/builder-bricks.component.html

```diff
@@ -1,4 +1,24 @@
 <ul class="builder-bricks">
-  <li class="builder-bricks__item" *ngFor="let brick of bricks; let i=index"><button
-      class="builder-bricks__delete" (click)="deleteElement(brick)">&times;</button></li>
+  <li
+    ngx-colors-trigger
+    class="builder-bricks__item"
+    *ngFor="let brick of bricks; let i = index"
+    [style.background]="color"
+    [(ngModel)]="color"
+    name="color"
+  >
```

Comment:
старайся сортировать аттрибуты тэгов
```suggestion
  <li
   *ngFor="let brick of bricks; let i = index"
    ngx-colors-trigger
    [(ngModel)]="color"
    [style.background]="color"
    name="color"
    class="builder-bricks__item"
  >
```